source/Missions/Services/Network/LiteNetP2PClient.cs

```diff
@@ -246,6 +246,11 @@ public void OnPeerDisconnected(NetPeer peer, DisconnectInfo disconnectInfo)
                 var peerDisconnectedEvent = new PeerDisconnected(peer, disconnectInfo);
                 messageBroker.Publish(this, peerDisconnectedEvent);
             }
+            else if (PeerServer != null)
+            {
+                ServerDisconnected serverDisconnected = new ServerDisconnected();
+                messageBroker.Publish(this, serverDisconnected);
+            }
```

Comment:
The following would be more explicit
```suggestion
                messageBroker.Publish(this, peerDisconnectedEvent);
                return;
            }
            if (peer == PeerServer)
            {
                ServerDisconnected serverDisconnected = new ServerDisconnected();
                messageBroker.Publish(this, serverDisconnected);
                return;
            }
```